WombatProject/Services/GoogleBooksAPIService.cs

```diff
@@ -8,88 +8,43 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 using WombatLibrarianApi.Models;
+using WombatLibrarianApi.Settings;
 
 namespace WombatLibrarianApi.Services
 {
     public class GoogleBooksAPIService : IBookAPIService
     {
-        private readonly IConfiguration _configuration;
-        public WombatBooksContext Context { get; }
-        public List<Book> AuthorBookItems { get; set; } = new List<Book>();
-        public List<Book> SearchResults { get; set; } = new List<Book>();
+        protected readonly GoogleApiSettings _googleApiSettings;
```

Comment:
Can't _googleApiSettings be private?

Reply:
Same as with _context, I thought it's cool to make it protected and readonly, thus it works as a private field now, but should we have a class inheriting GoogleBooksApiService, that class would have access to _googleApiSettings. I think both solutions should be good.